openslides_backend/action/actions/poll/anonymize.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Dict, List
+
+from ....models.models import Poll
+from ....services.datastore.commands import GetManyRequest
+from ....shared.patterns import Collection, FullQualifiedId
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionPayload
+from ..vote.remove_user_id import VoteRemoveUserId
+
+
+@register_action("poll.anonymize")
+class PollAnonymize(UpdateAction):
+    """
+    Action to anonymize a poll.
+    """
+
+    model = Poll()
+    schema = DefaultSchema(Poll()).get_update_schema()
+
+    def get_updated_instances(self, payload: ActionPayload) -> ActionPayload:
+        for instance in payload:
+            option_ids = self._get_option_ids(instance["id"])
+            options = self._get_options(option_ids)
+
+            for option_id in options:
+                option = options[option_id]
+                if option.get("vote_ids"):
+                    self._remove_user_id_from(option["vote_ids"])
+            if False:
+                yield instance
```

Comment:
This looks just wrong. If this is because mypy wants you to yield something, I would suggest to `return []` at the very end of the function

Reply:
Okay using `return []` is better. mypy wants a return value, so I used that hack. But with this return it works too.